packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -107,7 +107,7 @@ export function Drawer({
       open={true}
       onClose={onClose}
       placement="right"
-      getContainer={'.main-view'}
+      getContainer={'main'}
```

Comment:
Since the ExtensionSidebar is nested inside the `.main-view`, we can not continue using `.main-view` here, unfortunately.

However, this does reflect the purpose of the change, since we want to move the Drawer to the left of the Sidebar and below topnav.